layouts/Default.vue

```diff
@@ -1,7 +1,10 @@
 <template>
   <div class="default-layout">
     <overlay v-if="overlayActive" />
-    <loader />
+    <loader 
+      v-if="$store.state.ui.loader"
```

Comment:
`v-if` doesn't work on functional components. Can you check it with `$store.state.ui.loader === false` and `$store.state.ui.loader === true`